.github/workflows/test.yml

```diff
@@ -0,0 +1,68 @@
+name: 🧪 Test
+
+on: 
+  push:
+    branches:
+      - main
+      - dev
+  pull_request_target:
+
+jobs:
+  authorize:
+    environment: 
+      ${{ github.event_name == 'pull_request_target' &&
+      github.event.pull_request.head.repo.full_name != github.repository &&
+      'external' || 'internal' }}
+
+    runs-on: ubuntu-latest
+    steps:
+    - run: echo ✓
+  
+  playwright:
+    needs: authorize
+    name: 🎭 Playwright
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - name: 🛑 Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.11.0
```

Comment:
Can we check if this is scoped to the workflow. I just want to make sure it doesn't cancel a deploy workflow that is running as this workflow starts.

Reply:
It is scoped to a workflow according to https://github.com/marketplace/actions/cancel-workflow-action and there's particular configuration to stop other workflows.